src/main/resources/fields.json

```diff
@@ -0,0 +1,3131 @@
+[
```

Comment:
This is what I mean by unmanageable. End of the day this is a big form. But we can do better than an array of this many fields.

The absolute paths suggest a structure we can start with. But I dont think we need to stick to it religiously. As we get more familiar with the form we can experiment

Reply:
One strategy would be to develop a mapping from intuitive field names to the corresponding technical blob(s) you've got below.